remme/rest_api/transaction.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 REMME
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+import base64
+from contextlib import suppress
+
+from google.protobuf.message import DecodeError
+from sawtooth_sdk.protobuf.transaction_pb2 import Transaction
+
+from remme.account.client import AccountClient
+
+
+def post(payload):
+    tr = payload['transaction']
+    with suppress(Exception):
+        tr = tr.encode('utf-8')
+
+    try:
+        transaction = base64.b64decode(tr)
+    except Exception:
+        return {'error': 'Decode payload of tranasaction failed'}, 500
```

Comment:
Maybe error code 400? Same for transaction proto.

Reply:
Agree